hedera-node/hedera-app/src/main/java/com/hedera/node/app/tss/TssBaseServiceImpl.java

```diff
@@ -225,4 +227,69 @@ public void unregisterLedgerSignatureConsumer(@NonNull final BiConsumer<byte[],
     public TssHandlers tssHandlers() {
         return tssHandlers;
     }
+
+    @Override
+    @NonNull
+    public Roster chooseRosterForNetwork(
+            @NonNull State state,
+            @NonNull InitTrigger trigger,
+            @NonNull ServiceMigrator serviceMigrator,
+            @NonNull ServicesSoftwareVersion version,
+            @NonNull final Configuration configuration) {
+        if (!configuration.getConfigData(TssConfig.class).keyCandidateRoster()) {
+            final var readablePlatformStateStore =
+                    new ReadablePlatformStateStore(state.getReadableStates(PlatformStateService.NAME));
+            return buildRoster(requireNonNull(readablePlatformStateStore.getAddressBook()));
```

Comment:
TSS-Roster should deliver before TSS-Ledger-ID.  Once TSS-Roster delivers, there will be no more usage of `AddressBook` in the platform state.

Reply:
Agreed. We can change that when the PR gets merged for Rosters. Currently the `PlatformStateStore` doesn't provide Roster API.

Reply:
Added FUTURE comment to remove when roster service is implemented